### PR TITLE
sequencing cases conversion/download

### PR DESCRIPTION
**Pull request by Mathieu-Deharbe**

### CODE REVIEW

File: src/components/utils/caseUtils.ts [hunk 1]
```diff
@@ -200,19 +208,38 @@ export function useDownloadUtils() {
...
+            ) {
+                message += buildPartialDownloadMessage(
                     cases.length,
                     selectedElements
-                ),
-            });
+                );
+            }
+            if (message.length > 0) {
+                snackInfo({
```
Comment (thangqp) [inline src/components/utils/caseUtils.ts:239]: A small bug arrives after migrate the new version common ui, i.e. the snackInfo without headerId do not show the snack bar. headerId is optional as declared in the type definition file.
This bug will be corrected later in common ui

**Review by thangqp — APPROVED** [review]
Code OK, Test OK

File: src/components/utils/caseUtils.ts [hunk 2]
```diff
@@ -31,48 +32,48 @@ const downloadCases = async (uuids: string[]) => {
...
+            link.href = href;
+            link.setAttribute('download', filename);
+            document.body.appendChild(link);
+            link.click();
+            document.body.removeChild(link);
+        } catch (error: any) {
+            if (error.name === 'AbortError') {
+                throw error;
+            }
+            handleCaseExportError?.(caseElement, error);
```
Comment (jamal-khey) [inline src/components/utils/caseUtils.ts]: why do use use '?' here ? handleCaseExportError seem to be always defined

may be move handleCaseExportError definition on the top of the function ?

**Review by jamal-khey — APPROVED** [review]
Test OK
No warning on the console

File: src/components/utils/caseUtils.ts (hunk 2 above)
Comment (Mathieu-Deharbe) [reply src/components/utils/caseUtils.ts]: Indeed. Done.

File: src/components/utils/caseUtils.ts (hunk 1 above)
Comment (Mathieu-Deharbe) [reply src/components/utils/caseUtils.ts:239]: Indeed. But I merge anyway : this is a priority correction and it still work despite the bug in main. The snackbar is simply not displayed at the end of the download (or its cancellation).

**Pull request merged**